mailers/password_mailer/password_mailer.go

```diff
@@ -1,9 +1,105 @@
 package password_mailer
 
-// TODO: ドメイン取得後にSESによる送信を行う
-// https://github.com/sourcegraph/go-ses
-func Reset(email string, token string) {
+import (
+	"../../config"
+	"../../modules/logging"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+)
+
+func Reset(id int64, address string, token string) {
+	domain := config.Element("fqdn").(string)
+	resetURL := fmt.Sprintf("http://%s/passwords/%d/edit?token=%s", domain, id, token)
+	title := "Password reseted"
+	rawBody := "Your Fascia's password was reseted. Please access to following URL, and set new password. \n " + resetURL + " \n This URL is valid for 24 hours."
+	htmlBody := "<p>Your Fascia's password was reseted.</p><p>Please access to following URL, and set new password. </p><p><a href='" + resetURL + "'>" + resetURL + "</a></p><p>This URL is valid for 24 hours.</p>"
+	resp, err := sendMail(address, title, htmlBody, rawBody)
+
+	if err != nil {
+		logging.SharedInstance().MethodInfo("PasswordMailer", "Reset", true).Errorf("send mail error: %v", err)
+		return
+	}
+	logging.SharedInstance().MethodInfo("PasswordMailer", "Reset").Debugf("send mail response: %v", resp)
+	logging.SharedInstance().MethodInfo("PasswordMailer", "Reset").Info("success to send mail")
+
+}
+
+func Changed(address string) {
```

Comment:
__[golint]__
* [golint] exported function Changed should have comment or be unexported